Cargo.toml

```diff
@@ -1,15 +1,25 @@
 [package]
 name = "twobit"
 version = "0.1.2"
```

Comment:
we need to bump the minor version because of API changes

Reply:
Agreed, needs to be changed to 0.2.0